src/index.ts

```diff
@@ -26,11 +26,10 @@ log += lottery.getDurations();
 log += '\n\n';
 log += `TOTAL DURATION ${lottery.totalDuration} IN MILLISECONDS.`
 
-if (!fs.existsSync(dir)){
-    fs.mkdirSync(dir);
-}
 const timestamp = new Date().toISOString();
-const fileName = `${dir}/lottery-results_${timestamp}.txt`;
-writeFile(fileName, log, () => console.log('Check result file ' + fileName));
+const fileName = `${filePrefix}${timestamp}`;
+const dataFormat = 'txt';
```

Comment:
Can we create an enum for file extensions? I think it would simplify the process, the file exporter should know which formats it accepts by the enum.